run/hello-broken/README.md

```diff
@@ -0,0 +1,47 @@
+# Cloud Run Broken Sample
+
+This sample presents broken code in need of troubleshooting. View [improved](main.py#L43) for a more stable implementation. 
+
+Troubleshoot this code by following the [Local Container Troubleshooting Tutorial](http://cloud.google.com/run/docs/tutorials/local-troubleshooting).
+
+[![Run in Google Cloud][run_img]][run_link]
+
+[run_img]: https://storage.googleapis.com/cloudrun/button.svg
+[run_link]: https://console.cloud.google.com/cloudshell/editor?shellonly=true&cloudshell_image=gcr.io/cloudrun/button&cloudshell_git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&cloudshell_working_dir=run/hello-broken
+
+## Build
+
+```
+docker build --tag hello-broken:python .
+```
+
+## Run Locally
+
+```
+docker run --rm -p 9090:8080 hello-broken:python
+```
+
+## Test
+
+```
+pytest
```

Comment:
If they're cloning this entire repo, we have them run tests w/ nox, which does run pytest under the hood. The command would be `nox -s "py36(sample='./run/hello-broken')"`